src/iotjs_binding.cpp

```diff
@@ -85,49 +71,39 @@ JObject::JObject(bool v) {
 
 JObject::JObject(int v) {
   _obj_val = JVal::Number(v);
-  _unref_at_close = false;
+  _unref_at_close = true;
```

Comment:
Is there a case where _unref_at_close is false? If not, just remove this variable.